elastic_transport/_models.py

```diff
@@ -180,26 +188,24 @@ def hide_auth(val: str) -> str:
 
 @dataclass
 class ApiResponseMeta:
-    """Metadata that is returned from Transport.perform_request()"""
+    """Metadata that is returned from Transport.perform_request()
+
+    :ivar int status: HTTP status code
+    :ivar str http_version: HTTP version being used
+    :ivar HttpHeaders headers: HTTP headers
+    :ivar float duration: Number of seconds from start of request to start of response
+    :ivar NodeConfig node: Node which handled the request
+    :ivar typing.Optional[str] mimetype: Mimetype to be used by the serializer to decode the raw response bytes.
```

Comment:
I wasn't able to get Sphinx to show the `#:` comments, even with [autodata](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autodata), so I placed the information in the class docstring, using instance variables.